_posts/2015-06-29-donations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Have extra wireless equipment? Donate it to Meta Mesh!
+layout: post
+date: 2015-06-29
+author: Adam Longwill
+---
+
+{:.img}
+![Donate your WiFi equipment to Meta Mesh!](http://i.imgur.com/HY0Hrpy.jpg)
+
+If you're like us, you probably have some old computer equipment collecting cobwebs in your basement. Technology "graveyards"
+are a surefire symptom of being chronically sick with IT-itis, which is, of course, incurable.
+
+Fortunately Meta Mesh is anouncing a donation program that will allow you to donate your old wireless
+radios, routers, antennas, cables, and tools to Meta Mesh so that they can be given away for FREE to node hosts on PittMesh. Even
+though Meta Mesh's equipment is relatively inexpensive compared to many other wireless solutions, the cost to buy, configure, 
+and mount the equipment can still be prohibitive. Why not take up a collection and reuse old wireless equipment that is compatible with our routers?
+
+Here's a few details:
+    * Meta Mesh is not a 501c3 organization and therefore, your donations are NOT tax deductible.
```

Comment:
maybe add in here something like "if you want to target your donation to a specific charity operating in a neighborhood or municipality in which you would like to see PittMesh have a presence, we can work with you to make that happen"